rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java

```diff
@@ -788,8 +787,8 @@ public void finishNodeSourceShutdown(Client initiator) {
      * @return a list of down nodes
      */
     @ImmediateService
-    public LinkedList<Node> getDownNodes() {
-        LinkedList<Node> downNodes = new LinkedList<>();
+    public LinkedList<RMNode> getDownNodes() {
+        LinkedList<RMNode> downNodes = new LinkedList<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "downNodes" which hides the field declared at line 143. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)